docs/user-guide/replicasets.md

```diff
@@ -36,6 +36,9 @@ provides declarative updates to pods along with a lot of other useful features.
 Therefore, we recommend using Deployments instead of directly using Replica Sets, unless
 you require custom update orchestration or don't require updates at all.
 
+This actually means that you should never need to write a replica set object:
```

Comment:
We can't expect users to read every sentence carefully. Some possible ways to make it more obvious:

1. Bold the sentence
2. Create a sub-section titled something like "Manage via Deployment"
3. Create a sub-section for alternatives, add a section for Deployments and titled "Deployment (recommended)"
4. Modify the example to create a Deployment instead of a Replica Set

@bgrant0607 @madhusudancs WDYT?